src/main/java/fi/hsl/transitdata/metroats/models/MetroEstimate.java

```diff
@@ -0,0 +1,13 @@
+package fi.hsl.transitdata.metroats.models;
+
+import java.util.List;
+
+public class MetroEstimate {
+    public int schemaVersion;
```

Comment:
What is `schemaVersion`?

Reply:
I guess it ended up here because I copied MetroEstimate field from protobuf message.. Removing this now